test/e2e/serial/workload_placement_test.go

```diff
@@ -429,6 +429,124 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 			}, time.Minute, time.Second*5).Should(BeTrue(), "resources not restored on %q", updatedPod.Spec.NodeName)
 		})
 	})
+	Context("with at least one node with at least two numa zones", func() {
+		var nrtCandidates []nrtv1alpha1.NodeResourceTopology
+
+		BeforeEach(func() {
+			const requiredNUMAZones = 2
+			By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", requiredNUMAZones))
+			nrtCandidates := e2enrt.FilterZoneCountEqual(nrts, requiredNUMAZones)
```

Comment:
will FilterZoneCountEqual get all nodes exactly with 2 numa zones ? or above 2  ?  if the answer is the former then the candidates will not fit the test description

Reply:
`FilterZoneCountEqual` gets those nodes with exactly the number of zones given. For the test any node with at least two numa zones will fit ... but it seems we have some restrictions right now 

https://github.com/jlojosnegros/numaresources-operator/blob/8718cfa681abd706e56cf55bec8ef7f457d5342b/test/e2e/serial/workload_placement_test.go#L105

I do not know if it is worthy to repeat the comment on each call or it is better to make a general TODO comment in the file to avoid future problems.

Reply:
The best option seems to fix the text in the `By` above (remove "at least")